blocks/card/card.js

```diff
@@ -0,0 +1,34 @@
+import { getRelativePath, getIndexData } from '../../scripts/utils.js';
+
+export function populateCard(container, cardInfo) {
+  const card = document.createElement('div');
+  card.className = 'card';
+  card.innerHTML = `
+        <a href="${cardInfo.path}">
+        <picture>
+            <source srcset="${cardInfo.image}" media="(min-width: 750px)">
```

Comment:
Instead of srcset, can we replace with "createOptimizedImage" to get the webp (smaller) image size? See
https://github.com/hlxsites/ustglobal/blob/main/blocks/cards/cards.js#L18